pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -242,12 +242,21 @@ func getSmartRouterTemplate(cr *v1.KieApp) v1.SmartRouterTemplate {
 			template.KeystoreSecret = cr.Spec.Objects.SmartRouter.KeystoreSecret
 		}
 
+		if cr.Spec.Objects.SmartRouter.Protocol == "" {
+			template.Protocol = constants.SmartRouterProtocol
+		} else {
+			template.Protocol = cr.Spec.Objects.SmartRouter.Protocol
+		}
+
+		// if enpty defaults to false
```

Comment:
There's a typo with _empty_ but I think this comment is not needed as the default value for a boolean is `false`.